src/Post.php

```diff
@@ -513,8 +513,7 @@ public function reading_time(): ?int {
 	 */
 	public static function reading_time_block( array $attributes, $content, $block ) {
 		$time = ( new self( $block->context['postId'] ?? null ) )->reading_time();
-
-		return "<span class='article-list-item-readtime'>$time min read</span>";
+		return "<span class='article-list-item-readtime'>" . ($time ? $time . 'min read' : '') . "</span>";
```

Comment:
I've modified this logic in cases where `$time` is `null` and don't render unnecessarily `min read`

Reply:
Maybe instead of returning the span, if there's no `$time` we could return nothing? Or an empty string?